Examples/Python/src/ModuleEntry.cpp

```diff
@@ -102,53 +101,53 @@ PYBIND11_MODULE(ActsPythonBindings, m) {
     mv.attr("commit_hash_short") = Acts::CommitHashShort;
   }
 
```

Comment:
You could do `using namespace ActsPython` inside this function for conciseness.

Reply:
Done for the moment.